hack/dev-env/start-e2e2.sh

```diff
@@ -0,0 +1,53 @@
+#/bin/bash
+
+# getExternalLoadBalancerIP will set EXTERNAL_IP with the load balancer hostname from the specified Service
+getExternalLoadBalancerIP() {
+  SERVICE_NAME=$1
+
+  MAX_ATTEMPTS=120
+
+  for ((i=1; i<=MAX_ATTEMPTS; i++)); do
+    
+    echo ""
+    EXTERNAL_IP=$(kubectl get svc $SERVICE_NAME $K8S_CONTEXT $K8S_NAMESPACE -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
```

Comment:
On my local system, the hostname field is not set for the ingress. It looks like the following:

```
kubectl --context vcluster-control-plane get svc -n argocd argocd-redis -o jsonpath='{.status.loadBalancer}' | jq
{
  "ingress": [
    {
      "ip": "192.168.57.221",
      "ipMode": "VIP"
    }
  ]
}
```
We probably should accommodate both, ip and hostname cases.